### ISSUE TITLE
Triggering amplify hosting build using github actions

### ISSUE THREAD

**Issue by HardikGaur** [aws-amplify/amplify-hosting#3951]
I wanted to trigger amplify hosting builds for my nextjs website using github actions. I searched for user triggered build and came to this issue which mentions using webhooks to do this.
              Hi @dmudro that is correct, resolving this one for now, let us know if you have any issues getting that going

_Originally posted by @kahdojay in https://github.com/aws-amplify/amplify-hosting/issues/929#issuecomment-669301341_

Can we use github actions to trigger builds?

**Comment by mauerbac** [comment]
Yes, you can use manual webhooks to trigger builds -> https://docs.aws.amazon.com/amplify/latest/userguide/webhooks.html
